test/overlook-testing.js

```diff
@@ -31,6 +31,11 @@ describe('Get a customer\'s name from their username', () => {
     expect(user).to.equal('Please enter a valid username and password');
   });
 
+  it('Should know if user id is in the system', () => {
+    user = checkUsername('customer51');
+    expect(user).to.equal('Please enter a valid username and password');
+  });
+
```

Comment:
Also running into a bug with `customer0` and negative numbers in the username - sorry I should have tested that in the first go round

Reply:
it seems so obvious...